src/checks_linux.go

```diff
@@ -57,6 +63,11 @@ func firewallUp() (bool, error) {
 	return command("ufw status | grep -q 'Status: active'")
 }
 
+func passwordChanged(hash string, user string) (bool, error) {
+	res, err := fileContains("/etc/shadow", hash)
```

Comment:
I do really like the simplicity, but if someone copy and pastes a password with the same salt/hash to another user (or the password creation tool is faulty) then this check would fail (ex. john and bob both have password hash ABCD, change johns password and don't get points). Due the unlikeliness of this scenario, the simple code might be better,  but it is an edge case unhandled.